packages/proto-build/src/code_generator.rs

```diff
@@ -12,10 +12,12 @@ use crate::{mod_gen, transform};
 
 const DESCRIPTOR_FILE: &str = "descriptor.bin";
 
+#[derive(Clone)]
 pub struct CosmosProject {
     pub name: String,
     pub version: String,
     pub project_dir: String,
+    pub include_mods: Option<Vec<String>>,
```

Comment:
logically,`None == Some(vec![])` so if we aren't intended to use either of them separately, `Vec<String>`should just work.